cvat/apps/quality_control/serializers.py

```diff
@@ -81,6 +81,7 @@ class Meta:
             "max_validations_per_job",
             "iou_threshold",
             "oks_sigma",
+            "use_image_space_for_point_group_comparisons",
```

Comment:
can we find a better name for the parameter? 43 symbols and the whole sentence looks too much.